app/uk/gov/hmrc/plasticpackagingtax/registration/services/CountryService.scala

```diff
@@ -29,6 +30,7 @@ object FcoCountry {
 
 @Singleton
 class CountryService {
```

Comment:
```suggestion
class CountryService with Logging {
```
gives you a logger liek this for free :)